harvard-lts/drs-translation-service
# Hdc 44 Implement basic STOMP connection to DRS Queue

**Adds a STOMP listener to listen to the DRS Queue*
* * *

**GitHub Issue**: [(link)](https://github.com/harvard-lts/HDC/issues/58)

# What does this Pull Request do?
Connects to the DRS Queue and listens to the DRS_UPDATE_QUEUE topic.

# How should this be tested?

1. Clone the repo
2. Checkout the branch HDC-44_queuing_investigation
3. Copy the env-template.txt to .env
4. Get the credentials from LPE Shared-DRS/DEV-message-dev-ActiveMQ for STOMP
5. Add the credentials from step 4 to the .env for the DRS queueing (PROCESS is not needed for this)
6. Set .env variable `DRS_QUEUE_NAME=Consumer.librarycloud-dev.VirtualTopic.DRS_OBJECT_UPDATED`
7. Start up the docker container: `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
8. Exec into the container: `docker exec -it mqutils bash`
9. Run the command `pytest`
10. Only one test exists and it should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? Yes
